web/src/components/nav_bar/nav_options.js

```diff
@@ -4,50 +4,53 @@ class NavOption {
   constructor(displayName, url, isCurrentPageFunction) {
     this.displayName = displayName;
     this.url = url;
-    this.isCurrentPage = isCurrentPageFunction ?? this.defaultIsCurrentPage;
-  }
-
-  defaultIsCurrentPage() {
-    return window.location.hash === this.url;
+    const defaultIsCurrentPage = () => window.location.hash === this.url;
```

Comment:
why did you do this

Reply:
fair. Can you at least use better whitespacing?

Alternatively, javascript classes now support private methods:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties#private_methods